examples/cuda/schedulers/detail/reduce.cuh

```diff
@@ -73,7 +73,7 @@ namespace reduce_ {
 
         static constexpr std::size_t value =
           _P2300::__v<
-            std::execution::__gather_sigs_t<
+            _P2300::execution::__gather_sigs_t<
               std::execution::set_value_t, 
```

Comment:
I believe that should also be `_P2300::execution::set_value_t`

Reply:
I don't think you're commenting on the most recent commit. "`_P2300`" isn't a thing after this PR.